### PR TITLE
fix(Nav): Center title, add mobile hamburger menu

### PR DESCRIPTION
**Pull request by dunphyben**

### CODE REVIEW

File: src/layouts/Nav/nav.module.styl [hunk 1]
```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
...
   padding-right $navbar-padding-vertical
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
+
+
+.hamburger
+  flex 1
+  align-self center
+  @media (min-width: 760px)
```
Comment (BerkeleyTrue) [inline src/layouts/Nav/nav.module.styl]: This should use the `$above` function and the breakpoints set in the scale function

above func https://github.com/realworldreact/jamcommerce/blob/master/src/functions.styl#L142

scale https://github.com/realworldreact/jamcommerce/blob/master/src/vars.styl#L67

File: src/layouts/Nav/nav.module.styl [hunk 2]
```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
   padding-left $navbar-padding-vertical
   padding-right $navbar-padding-vertical
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
```
Comment (BerkeleyTrue) [inline src/layouts/Nav/nav.module.styl]: This class already has flex, if you want to justify the content then pass the term 'between' as the first arg to `$grid`

File: src/layouts/Nav/nav.module.styl [hunk 3]
```diff
@@ -56,3 +70,17 @@ $navbar($height, $font-size = 20px)
...
     a
       $column(center, center)
+      text-transform uppercase
+      letter-spacing 2.75px
+      color $menu-unselected
+      font-weight 100
+      margin-left 17px
+      margin-right 17px
+      padding-left 0
+      padding-right 0
```
Comment (BerkeleyTrue) [inline src/layouts/Nav/nav.module.styl]: css attributes should be alphabetically sorted

File: src/layouts/Nav/nav.module.styl [hunk 4]
```diff
@@ -36,14 +36,28 @@ $navbar($height, $font-size = 20px)
...
   border-bottom 1px solid $gray
+  display flex
+  justify-content space-between
+
+
+.hamburger
+  flex 1
+  align-self center
+  @media (min-width: 760px)
+    visibility hidden
```
Comment (BerkeleyTrue) [inline src/layouts/Nav/nav.module.styl]: use display none instead of visibility

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

File: src/layouts/Nav/Nav.jsx [hunk 5]
```diff
@@ -14,21 +15,26 @@ export default function Nav() {
...
-              Sign In
-            </a>
-          </li>
-          <li>
-            <a className={ cx('cart') }>
-              <img src={ cart } />
-            </a>
-          </li>
-        </ul>
+        <div className={ cx('account') }>
```
Comment (BerkeleyTrue) [inline src/layouts/Nav/Nav.jsx]: Instead of wrapping this in a div just add the class to the ul directly

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

File: src/layouts/Nav/nav.module.styl (hunk 4 above)
Comment (dunphyben) [reply src/layouts/Nav/nav.module.styl]: using display none removes the element from the DOM altogether, which means there are only 2 flex elements in the Nav. The result is the title is no longer centered.

Comment (BerkeleyTrue) [reply src/layouts/Nav/nav.module.styl]: ah, makes sense. Ignore this one

File: src/layouts/Nav/nav.module.styl (hunk 1 above)
Comment (BerkeleyTrue) [reply src/layouts/Nav/nav.module.styl]: You missed this one

**Pull request merged**